packages/server/test/e2e/2_controllers_spec.coffee

```diff
@@ -14,3 +14,12 @@ describe "e2e plugins", ->
       snapshot: true
       expectedExitCode: 1
     })
+
+  ## this tests open mode behavior
+  ## when the user hits the 'reload' button in open mode
+  ## https://github.com/cypress-io/cypress/issues/6619
+  e2e.it 'can reload during spec run', {
+    spec: "reload-spec.spec.js"
+    snapshot: true
+    timeout: 15000
+  }
```

Comment:
Why put this in the controllers spec?

Reply:
idk what a controller is in this case, figured "spec does not exit" was the closest thing to this. I could create a `1_runner_spec.js` though

Reply:
The controller facilitates serving the file to the browser, so the other test is concerned with the spec file not existing and how the spec controller handles that. 

Probably make sense to have this be an "issue" spec, like all the `3_issue_*` ones, since it's addressing a specific issue and isn't a feature or anything.